### DISCUSSION TITLE
AgentMonitoring and JobAgent entries in ES

### DISCUSSION THREAD

**Discussion by arrabito** [DIRACGrid/DIRAC#7021] [Q&A] [answered]
Dear all,

I've enabled the AgentMonitoring to use the Monitoring System and it works well. 
However, not only the agents running on the servers produce entries in the ES database, but also the JobAgent running on the worker nodes produces many entries in the ES database.
I don't think this is a wanted behaviour. Is there anyway to disable the MS for this agent in particular?

Se below an entry produced by the JobAgent.

Thank you.

```
AgentName:
    WorkloadManagement/JobAgent
timestamp:
    May 17, 2023 @ 08:53:34.443
Host:
    cn-315-02-01.cr.cnaf.infn.it
MemoryUsage:
    60.949
CpuPercentage:
    0.044
CycleDuration:
    360.491
_id:
    IUx8KIgBE-S6BSo5uGS5
_type:
    _doc
_index:
    cta-_agent_monitoring-index-2023-05
_score:
    - 
```

**Comment by fstagni** [comment, marked as answer]
I think it will be fixed by https://github.com/DIRACGrid/DIRAC/pull/7080
